openstack/02a-MAClow/IEEE802154E.h

```diff
@@ -35,14 +35,19 @@ static const uint8_t ebIEsBytestream[] = {
 #define EB_SLOTFRAME_LEN_OFFSET     20
 #define EB_SLOTFRAME_NUMLINK_OFFSET 22
 
-#define EB_IE_LEN                   28
+/**
+Set EB on minimal cell for 1/EB_PORTION portion. 
+Must be set to lower values for slower modulations. Otherwise, impossible to synchronize. 
+Default setting is 10 for 20 ms slot duration and 4 for 120 ms slot duration
+*/
+#define EB_PORTION                  4 

```

Comment:
This variable  is configurable, leave it as 10 by default. It influence the synchronization time, but still be able to synchronize with larger value.